docs/modules/ROOT/pages/guides/guides-configuration-store.adoc

```diff
@@ -3,16 +3,17 @@
 == Overview
 The Configuration Store is a tool that enables the various configurations for a project to be centrally defined and
 managed, while also standardizing access to the configurations. The Configuration Store dynamically provides
-environment-specific configurations, based on the project's development lifecycle phase. Usage limitations and
-regeneration strategies can also be set, allowing configurations to automatically refreshed, thereby bolstering
-security of sensitive properties.
```

Comment:
S (Substanative): I assume this is just the _first_ of the options, correct?  A Vault implementation, for instance, may well make sense for larger projects and/or test/prod use.  Should we segment this guide out a bit to account for that?

Reply:
Sorry not sure if I follow. This overview section doesn't talk about implementations specifics like vault vs in memory.  I will add a paragraph bellow talking about the current options.

Reply:
Yeah, that's what I'm getting at.  Since this in interface implementation, we probably want to radiate that concept into the docs.